core/src/main/scala/org/apache/spark/storage/BlockManager.scala

```diff
@@ -1074,7 +1074,7 @@ private[spark] class BlockManager(
    * Remove all blocks belonging to the given broadcast.
    */
   def removeBroadcast(broadcastId: Long, tellMaster: Boolean): Int = {
-    logInfo(s"Removing broadcast $broadcastId")
```

Comment:
Yeah that one is sufficient then. I looked earlier, but only at the `doDestroy` implementations which did not have info level logging.